main.c

```diff
@@ -24,19 +25,22 @@
 void main(void)
 {
     // stop watchdog timer
-	WDT_A->CTL = WDT_A_CTL_PW | WDT_A_CTL_HOLD;
+    WDT_A->CTL = WDT_A_CTL_PW | WDT_A_CTL_HOLD;
 
-	// Set clock to 24 MHz
-	set_DCO(FREQ_24_MHZ);
+    // Set clock to 24 MHz
+    set_DCO(FREQ_24_MHZ);

```

Comment:
Should be FREQ_48_HZ. Also @delaneyberger, can you change you #define CLK to FREQ_48_MHZ instead of FREQ_24_MHZ. You can just make this change in your next pull request.